platform/android/export/export.cpp

```diff
@@ -1831,6 +1833,25 @@ class EditorExportPlatformAndroid : public EditorExportPlatform {
 		return list;
 	}
 
+	inline bool is_clean_build_required(Vector<PluginConfig> enabled_plugins) {
+		bool first_build = last_custom_build_time == 0;
```

Comment:
If `first_build` is `true`, you can skip the rest of the logic below similarly to what you do when `have_plugins_changed` is `true`.

Reply:
Done.